packages/apps-config/src/endpoints/testing.ts

```diff
@@ -316,10 +316,10 @@ export function createTesting (t: TFunction, firstOnly: boolean, withSort: boole
       }
     },
     {
-      info: 'oak-testnet',
-      text: t('rpc.test.oak', 'OAK Testnet', { ns: 'apps-config' }),
+      info: 'neumann',
+      text: t('rpc.test.oak', 'Neumann Network', { ns: 'apps-config' }),
       providers: {
-        'OAK Network': 'wss://rpc.testnet.oak.tech'
+        'OAK Network': 'wss://node-6874600165100077056.jm.onfinality.io/ws?apikey=00e65cb5-0bcc-4dac-9a6d-08e6e111e186'
```

Comment:
we need to switch this to the api service onfinality provides, once we have a version we're comfortable with, then we can proceed

Reply:
We need to bind the actual API endpoint to the domain `rpc.testnet.oak.tech`, rather than exposing it, so this line of code shouldn’t be changed.

Reply:
Another reason is that the domain rpc.testnet.oak.tech is easier to remember.

Reply:
`wss://oak.api.onfinality.io/public-ws` We can get the api service from onfinality and bind it to that. We can add the rpc bind at a later time.

Reply:
Sounds good!

Reply:
How does it distinguish networks (OAK, Turing, Neumann) through one URL?

`wss://oak.api.onfinality.io/public-ws`

@irsal